docs/05-components/empty-state.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Empty State
+description: Use an empty state to communicate to the user that there is no data to display, or that a search query returned no results.
+draft: true
+---
+
+# EmptyState <Badge text='draft' color='orange'></Badge> <a href='https://developers.grafana.com/ui/latest/index.html?path=/story/general-emptystate--basic' target='_blank' className='header-links'>Storybook <Icon name="external-link-alt"/> </a>
+
+The EmptyState component consists of a message and optionally an image, button, and additional information.
+
+## When to use
+
+Use an empty state to communicate to the user that there is no data to display and provide instructions for what to do next.
+Example use cases:
+- When a user has not created a particular resource yet
+- When a filter or search query returns no results
+- When a user completes all actions, such as clearing their inbox or notifications
+
+There are different variants to handle the most common empty state use cases. These variants provide a specific default message and image.
+
+## Usage
+
+### `variant="call-to-action"`
+
+Use when there is no data to display and you want to encourage the user to take an action. Usually either to complete some initial configuration, or to create an item.
+
+```tsx
+import { EmptyState, LinkButton, TextLink } from '@grafana/ui';
+
+<EmptyState
+  variant="call-to-action"
```

Comment:
Add the `live` tag after the tsx language specifier (` \`\`\`tsx live `) and make this an interactive demo ✨ 

You can remove the import statement as well.

Reply:
don't think i can do that until it's released in `@grafana/ui` though 😬

Reply:
Ahh true. You can bump grafana-ui to canary version then?

Reply:
if we're happy with that then yeah i can do that 👍

Reply:
have updated to canary and added the live version of `not-found` (it's the only one that exists currently - waiting for an svg for the other variant 😭 )